docker/ds/Dockerfile

```diff
@@ -6,8 +6,8 @@ ARG VERSION="6.5.0-M127.1"
 RUN download -v $VERSION opendj
 
 # JDK 11 tracking - https://bugster.forgerock.org/jira/browse/OPENDJ-5207
-FROM openjdk:11-jdk 
-#FROM openjdk:8-jdk 
+#FROM openjdk:11-jdk 
+FROM openjdk:8-jdk 
```

Comment:
It would be really great to stop changing this in our master bits. Maybe folks could create a branch when they want to use JDK 11 in testing?